src/main/java/nl/novi/techiteasy/TechItEasyApplication.java

```diff
@@ -9,5 +9,4 @@ public class TechItEasyApplication {
     public static void main(String[] args) {
         SpringApplication.run(TechItEasyApplication.class, args);
     }
-//adding comment for feedback mark
```

Comment:
Hier valt weinig feedback aan te geven. Deze klasse is standaard en hoeven we ook niks aan te doen :)